server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/GetMessagesMethodStepdefs.java

```diff
@@ -235,6 +235,11 @@ public void appendMessageWithSpecialCase(String messageName, String mailbox) thr
         appendMessage(messageName, "eml/htmlWithLongAndComplicatedContent.eml");
     }
 
+    @Given("^the user has a message \"([^\"]*)\" in \"([^\"]*)\" mailbox with iso charset")
+    public void appendMessageWithEncoding(String messageName, String mailbox) throws Throwable {
+        appendMessage(messageName, "eml/iso8859_1charset.eml");
```

Comment:
we really don't need an resource to have a one line message, we should generate them.
That way, this method could take the charset as an argument and we could have a datatable for executing the test for a lot of charsets. WDYT ?